scripts/screens/EventEditScreen.py

```diff
@@ -2118,7 +2257,38 @@ def valid_supply(self) -> bool:
 
         return valid
 
+    def valid_future(self):
+        """
+        Checks if future blocks have all required info
+        """
+        valid = True
+        for block in self.future_block_list:
+            pool = block["pool"]
+            if not block["event_type"]:
+                valid = False
+                self.alert_text = (
+                    "A Future Event block has no type. A type must be chosen!"
+                )
+            elif (
+                not pool.get("subtype")
+                and not pool.get("event_id")
+                and not pool.get("excluded_event_id")
+            ):
+                valid = False
+                self.alert_text = "A Future Event block has no subtype, event_id, or excluded_event_id given. Event pool is too broad, you must use at least one of these constraints!"
+            elif block["moon_delay"][0] > block["moon_delay"][1]:
+                valid = False
+                self.alert_text = "A Future Event block has an invalid moon delay. The 2nd moon delay number should be equal to or larger than the 1st!"
```

Comment:
```suggestion
                self.alert_text = "A Future Event block has an invalid moon delay. The second moon delay number should be equal to or larger than the first!"
```
Unless this is a britishism I'm not aware of, you should always write out numbers unless it's a date (I think the threshold for this is twenty, but I might be wrong there...)